pkg/runner/runner.go

```diff
@@ -222,7 +222,19 @@ func Run(cfg *Config) error {
 		},
 	)
 
+	pendingTelemetry := map[string]string{
+		"recording_enabled": "false", // Will be written in telemetry table as `promscale_recording_enabled: false`
+		"alerting_enabled":  "false", // `promscale_alerting_enabled: false`
+	}
 	if cfg.RulesCfg.ContainsRules() {
+		pendingTelemetry["recording_enabled"] = "true"
```

Comment:
Again, we cannot check if it is a recording rules or alerting rules. We just know it is some rules. And from telemetry's perspective, we want to know if users are using rules or not. If there exists an alerting config, then there will obviously be a alerting rule, otherwise it makes no sense to have alerting config.